### PR TITLE
Responses FR

### PR DESCRIPTION
**Pull request by VanSlanzar**
Here is what I suggest, to be as close as possible to the translation. 
Does this document work for all extensions of FFXIV ?

**In English, does he suggest going behind a player, something or the boss?**

 Derriere boss

does this document work for all extensions of FFXIV

### CODE REVIEW

**Comment by quisquous** [comment]
I think this is only used for "get behind the boss", e.g. e8 biting frost.  Things like rocks usually get "hide behind boulder/door/meteor" style text.

> does this document work for all extensions of FFXIV

Sorry, could you rephrase what you're asking? I'm not sure I follow.

**Comment by VanSlanzar** [comment]
I was talking about files (Response .js), it's for all extensions ?

**Comment by quisquous** [comment]
> I was talking about files (Response .js), it's for all extensions ?
> if it is yes I would suggest verbs or expressions, but I don't know how to format

Sorry for my ongoing confusion, but I still don't understand what you're asking.  I think I don't understand the word "extensions" here, and so I don't understand your question.

**Comment by VanSlanzar** [comment]
(ARR, HwS, StB, ShB) FFXIV all game = all extension in French ^^.

Do not worry, I understand that it is not always easy to understand each other.

**Comment by quisquous** [comment]
Ah! Thanks. In English, those are "expansions".  And yes, Responses.js applies to all of them.  Did you have suggestions of missing responses that would help?  My thought process was to add responses that were used many times, but any response that was fight-specific would just be in the trigger file itself.

**Comment by VanSlanzar** [comment]
YES can I put my proposals like this:
- en: Get behind
   fr: Allez derrière
I do not think I am competent to put it otherwise.

**Comment by quisquous** [comment]
> YES can I put my proposals like this:
> 
> * en: Hide Behind
>   fr: Cachez-vous derrière
>   I do not think I am competent to put it otherwise.

Feel free to either write it out as a comment, or propose via a code change in the pull request.  Whatever is convenient.

> Do not validate This request, because:

Ok, will wait for other people to answer here, thanks!

**Comment by VanSlanzar** [comment]
OK, I have a bilingual friend who confirmed to me (Get Behind)> (Go behind). I correct my error, my apologies.

**Comment by mooondark** [comment]
_Passez_ or _Allez_ are valid. I think _Allez_ was more used when I started to help translation.

**Comment by VanSlanzar** [comment]
there is a difference in definition. I find (pass) better to adapt. now that i'm sure everything is fine. I think I will often disturb my friend ^^. TY @moondark, and @quisquous. I will keep asking when I have doubts.

**Comment by mooondark** [comment]
Franchement, ne soit pas si pointilleux. Que ce soit "Passez" ou "Allez", à la fin, tu es normalement derrière. Te prends pas la tête comme ça. De mon avis, ce n'étais pas la peine de changer, maintenant c'est fait ce n'est pas grave.
Tu peux m'ajouter sur discord si tu veux moyse#2147

**Comment by VanSlanzar** [comment]
avec grand plaisir.

**Comment by quisquous** [comment]
Is this ok to merge or did you want to make any additional changes based on the conversation above?

**Comment by VanSlanzar** [comment]
it's ok to merge. We speak with @moondark on discord for other replacment. TY
